addons/web/static/src/js/views/form_upgrade_widgets.js

```diff
@@ -23,7 +23,7 @@ var AbstractFieldUpgrade = {
         this._super.apply(this, arguments);
         this.get_enterprise_label().after($("<span>", {
             text: "Enterprise",
-            'class': "label label-primary oe_inline"
+            'class': "label label-primary oe_inline oe_enterprise_label"
```

Comment:
This class is used for the label "Enterprise" on settings available only in enterprise edition. It does not appear in enterprise, only in community.